maven-java/kalix-spring-boot-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -146,6 +146,7 @@
           <arguments>
             <argument>-classpath</argument>
             <classpath/>
+            <argument>-Dlogging.config=src/main/resources/logback-dev-mode.xml</argument>
```

Comment:
Spring takes control of the logging infra and tries to have a single property for all log frameworks. Instead of passing `logback.configurationFile`, we should pass `logging.config` and with the full path to the file.